fcrepo-http-api/src/test/resources/test-objects/test.rdf

```diff
@@ -0,0 +1,123 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Good point - I simply included RDF in the test that I _knew_ always caused an exception.. but that aspect is not important for this test.  The point of the test is just to make sure that we get back exactly what we put in, for which a one liner is sufficient.